services/core-web/src/components/homepage/HomeSidePanel.tsx

```diff
@@ -63,6 +63,10 @@ const ExternalLinksSection = () => {
       title: "First Nation Consultation System",
       url: "https://apps.nrs.gov.bc.ca/int/fncs",
     },
+    {
+      title: "Regional Mines Public Engagement Tool Portal",
+      url: "http://www.gov.bc.ca/minesengagement",
```

Comment:
Code scanning tools do favour https over http- when I copied this link into my browser it redirected me to an HTTPS url, and even just adding in the "s" to this link worked. Would recommend probably adding the "s"

Reply:
Good eye, this has been changed now.